### PR TITLE
fix(CN-513): all docs=true suffixed links after zendesk migration

### PR DESCRIPTION
**Pull request by beths-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - Navigate to the updated links. The links are no longer broken

### 📝 Description

See https://ledger.slack.com/archives/C02PQA777EU/p1732181750558089

Staking links such as https://support.ledger.com/article/360014339340-Earn-Cosmos-ATOM-staking-rewards-in-Ledger-Live?docs=true-zd?redirect=false
were broken in the zen desk migration. 

- Links are fixed, and any that suffered the same problem because of the query parameters `?docs=true` are also now directing to what appears to be the correct articles again.
- Also added --runInBand and maxWorkers 50% to mobile tests to see if we could improve reliability of tests. See https://jestjs.io/docs/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server
- This should mean that the CPUs do not max out, which is a common cause of random "timeout" fails on CI.

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/CN-513

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by LucasWerey — APPROVED** [review]

File: apps/ledger-live-mobile/e2e/jest.config.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const jestAllure2ReporterOptions = {
 
 module.exports = async () => ({
   rootDir: "..",
-  maxWorkers: process.env.CI ? 2 : 1,
+  maxWorkers: "50%",
```
Comment (abdurrahman-ledger) [inline apps/ledger-live-mobile/e2e/jest.config.js:24]: This reduces emulator/simulator number to one on the CI. Are we sure we want this ?
Could also be the reason why test are more stable (less ressources used with only one device)

File: apps/ledger-live-mobile/scripts/e2e-ci.mjs [hunk 2]
```diff
@@ -61,6 +62,7 @@ const test_android = async () => {
     --forceExit \\
     --headless \\
     --retries 1 \\
+    --runInBand \\
```
Comment (abdurrahman-ledger) [inline apps/ledger-live-mobile/scripts/e2e-ci.mjs:65]: Why applying this on Android also? As far as I know we do not have performance issue on Linux runners.

File: apps/ledger-live-mobile/e2e/jest.config.js (hunk 1 above)
Comment (beths-ledger) [reply apps/ledger-live-mobile/e2e/jest.config.js:24]: Ah, interesting, I wonder why. So would you prefer `process.env.CI ? "50%" : 1,` ?

File: apps/ledger-live-mobile/scripts/e2e-ci.mjs (hunk 2 above)
Comment (beths-ledger) [reply apps/ledger-live-mobile/scripts/e2e-ci.mjs:65]: interestingly the Android run failed for the first time if I only added it to the ios tests. It also shouldn't cause any issues to apply unilaterally, as far as I understand

**Review by ComradeAERGO — APPROVED** [review]

**Review by LucasWerey — APPROVED** [review]

**Pull request merged**